lib/asciidoctor/js/asciidoctor_ext/browser/abstract_node.rb

```diff
@@ -1,5 +1,29 @@
 module Asciidoctor
 class AbstractNode
+  def read_contents target, opts = {}
+    doc = @document
+    if (Helpers.uriish? target) || ((start = opts[:start]) && (Helpers.uriish? start) && (target = doc.path_resolver.web_path target, start))
+      if (doc.path_resolver.descends_from? target, doc.base_dir) || (doc.attr? 'allow-uri-read')
+        begin
+          if opts[:normalize]
+            (Helpers.prepare_source_string ::File.read(target).join LF)
```

Comment:
I'm just using `File.read(target)` instead of `::OpenURI.open_uri(target, URI_READ_MODE) {|f| [f.content_type, f.read] }`